packages/shared-types/src/ComponentThemeVariables.ts

```diff
@@ -619,6 +619,7 @@ export type ImgTheme = {
 }
 
 export type LinkTheme = {
+  borderRadius: Border['radiusSmall']
```

Comment:
Since this is applied to the focus ring, I would group it with the other "focusOutline" variables, and call it `focusOutlineBorderRadius`. It is always a good practice to name the variables in a way that it is obvious what they are for. 

Also, please merge the commits into one. :)